### ISSUE TITLE
how to test your pretrained model on a picture from my own computer

### ISSUE THREAD

**Issue by ZeyingXuHuaWei** [zhou13/neurvps#7]
hello， I am poor at programming,  can you tell me how to test your pretrained model on a picture from my own computer?

**Comment by zhou13** [comment]
Currently you need to write you own dataset to test it. I might provide a script for this in the future.

**Comment by ZeyingXuHuaWei** [comment]
@zhou13 thank you for your reply!   May I ask another question. Suppose that three vanishing points of a picture in manhattan world are detected, can we compute the camera pose (including position and rotation ) or  can we just obtain camera rotation?

**Comment by zhou13** [comment]
camera rotation.

**Comment by hashimK** [comment]
@zhou13 I tried to make my own dataset file by reverse-engineering the datasets.py file and realized that label files like .mat, .txt, .npz are needed for each image to test. 
But I really want to test any random image with no label data. Hence, request you to provide a script/example that shows how to get vanishing points of any image without having any label data. 
Thank you.

**Comment by zhou13** [comment]
@hashimK The easiest way to do that for now is to create a dataset with a random label. Then you will be able to test it and find the vanishing point. To finding the dominant vanishing point, I would suggest use the pre-trained model on Natural Scene dataset.

**Comment by hashimK** [comment]
@zhou13 Thanks! It helps this way

**Comment by xiaoxiaoyi689** [comment]
Hi Zhou! Could you provide the test script?

**Comment by zhou13** [comment]
Sorry, I still don't have the scripts. I will close this issue as it is unlikely for me to provide such a script in the near future. However, I will be happy to test one if somebody makes an PR.
